WARNING 2022-07-20 15:13:10 StatReload detected file change in 'apps/permission/views.py'. Reloading...
INFO 2022-07-20 15:13:11 Started server process [25117]
INFO 2022-07-20 15:13:11 Waiting for application startup.
INFO 2022-07-20 15:13:11 Application startup complete.
WARNING 2022-07-20 15:17:24 StatReload detected file change in 'apps/permission/curd/curd_role.py'. Reloading...
INFO 2022-07-20 15:17:24 Shutting down
INFO 2022-07-20 15:17:24 Waiting for application shutdown.
INFO 2022-07-20 15:17:24 Application shutdown complete.
INFO 2022-07-20 15:17:24 Finished server process [25117]
WARNING 2022-07-20 15:19:01 StatReload detected file change in 'apps/permission/curd/curd_role.py'. Reloading...
INFO 2022-07-20 15:19:02 Started server process [25230]
INFO 2022-07-20 15:19:02 Waiting for application startup.
INFO 2022-07-20 15:19:02 Application startup complete.
WARNING 2022-07-20 15:19:18 StatReload detected file change in 'apps/permission/curd/curd_role.py'. Reloading...
INFO 2022-07-20 15:19:18 Shutting down
INFO 2022-07-20 15:19:19 Waiting for application shutdown.
INFO 2022-07-20 15:19:19 Application shutdown complete.
INFO 2022-07-20 15:19:19 Finished server process [25230]
INFO 2022-07-20 15:19:19 Started server process [25238]
INFO 2022-07-20 15:19:19 Waiting for application startup.
INFO 2022-07-20 15:19:19 Application startup complete.
WARNING 2022-07-20 15:20:58 StatReload detected file change in 'apps/permission/curd/curd_role.py'. Reloading...
INFO 2022-07-20 15:20:58 Shutting down
INFO 2022-07-20 15:20:59 Waiting for application shutdown.
INFO 2022-07-20 15:20:59 Application shutdown complete.
INFO 2022-07-20 15:20:59 Finished server process [25238]
INFO 2022-07-20 15:20:59 Started server process [25262]
INFO 2022-07-20 15:20:59 Waiting for application startup.
INFO 2022-07-20 15:20:59 Application startup complete.
WARNING 2022-07-20 15:22:20 StatReload detected file change in 'apps/permission/curd/curd_role.py'. Reloading...
INFO 2022-07-20 15:22:20 Shutting down
INFO 2022-07-20 15:22:20 Waiting for application shutdown.
INFO 2022-07-20 15:22:20 Application shutdown complete.
INFO 2022-07-20 15:22:20 Finished server process [25262]
INFO 2022-07-20 15:22:21 Started server process [25293]
INFO 2022-07-20 15:22:21 Waiting for application startup.
INFO 2022-07-20 15:22:21 Application startup complete.
WARNING 2022-07-20 15:27:01 StatReload detected file change in 'apps/permission/curd/curd_role.py'. Reloading...
INFO 2022-07-20 15:27:01 Shutting down
INFO 2022-07-20 15:27:01 Waiting for application shutdown.
INFO 2022-07-20 15:27:01 Application shutdown complete.
INFO 2022-07-20 15:27:01 Finished server process [25293]
INFO 2022-07-20 15:27:02 Started server process [25371]
INFO 2022-07-20 15:27:02 Waiting for application startup.
INFO 2022-07-20 15:27:02 Application startup complete.
WARNING 2022-07-20 15:29:44 StatReload detected file change in 'apps/permission/curd/curd_role.py'. Reloading...
INFO 2022-07-20 15:29:44 Shutting down
INFO 2022-07-20 15:29:44 Waiting for application shutdown.
INFO 2022-07-20 15:29:44 Application shutdown complete.
INFO 2022-07-20 15:29:44 Finished server process [25371]
INFO 2022-07-20 15:29:45 Started server process [25415]
INFO 2022-07-20 15:29:45 Waiting for application startup.
INFO 2022-07-20 15:29:45 Application startup complete.
ERROR 2022-07-20 15:29:56 Exception in ASGI application
Traceback (most recent call last):
  File "/opt/homebrew/Caskroom/miniforge/base/lib/python3.9/site-packages/uvicorn/protocols/http/h11_impl.py", line 366, in run_asgi
    result = await app(self.scope, self.receive, self.send)
  File "/opt/homebrew/Caskroom/miniforge/base/lib/python3.9/site-packages/uvicorn/middleware/proxy_headers.py", line 75, in __call__
    return await self.app(scope, receive, send)
  File "/opt/homebrew/Caskroom/miniforge/base/lib/python3.9/site-packages/uvicorn/middleware/debug.py", line 96, in __call__
    raise exc from None
  File "/opt/homebrew/Caskroom/miniforge/base/lib/python3.9/site-packages/uvicorn/middleware/debug.py", line 93, in __call__
    await self.app(scope, receive, inner_send)
  File "/opt/homebrew/Caskroom/miniforge/base/lib/python3.9/site-packages/fastapi/applications.py", line 261, in __call__
    await super().__call__(scope, receive, send)
  File "/opt/homebrew/Caskroom/miniforge/base/lib/python3.9/site-packages/starlette/applications.py", line 119, in __call__
    await self.middleware_stack(scope, receive, send)
  File "/opt/homebrew/Caskroom/miniforge/base/lib/python3.9/site-packages/starlette/middleware/errors.py", line 181, in __call__
    raise exc
  File "/opt/homebrew/Caskroom/miniforge/base/lib/python3.9/site-packages/starlette/middleware/errors.py", line 159, in __call__
    await self.app(scope, receive, _send)
  File "/opt/homebrew/Caskroom/miniforge/base/lib/python3.9/site-packages/starlette/middleware/cors.py", line 92, in __call__
    await self.simple_response(scope, receive, send, request_headers=headers)
  File "/opt/homebrew/Caskroom/miniforge/base/lib/python3.9/site-packages/starlette/middleware/cors.py", line 147, in simple_response
    await self.app(scope, receive, send)
  File "/opt/homebrew/Caskroom/miniforge/base/lib/python3.9/site-packages/starlette/exceptions.py", line 87, in __call__
    raise exc
  File "/opt/homebrew/Caskroom/miniforge/base/lib/python3.9/site-packages/starlette/exceptions.py", line 76, in __call__
    await self.app(scope, receive, sender)
  File "/opt/homebrew/Caskroom/miniforge/base/lib/python3.9/site-packages/fastapi/middleware/asyncexitstack.py", line 21, in __call__
    raise e
  File "/opt/homebrew/Caskroom/miniforge/base/lib/python3.9/site-packages/fastapi/middleware/asyncexitstack.py", line 18, in __call__
    await self.app(scope, receive, send)
  File "/opt/homebrew/Caskroom/miniforge/base/lib/python3.9/site-packages/starlette/routing.py", line 659, in __call__
    await route.handle(scope, receive, send)
  File "/opt/homebrew/Caskroom/miniforge/base/lib/python3.9/site-packages/starlette/routing.py", line 259, in handle
    await self.app(scope, receive, send)
  File "/opt/homebrew/Caskroom/miniforge/base/lib/python3.9/site-packages/starlette/routing.py", line 61, in app
    response = await func(request)
  File "/opt/homebrew/Caskroom/miniforge/base/lib/python3.9/site-packages/fastapi/routing.py", line 227, in app
    raw_response = await run_endpoint_function(
  File "/opt/homebrew/Caskroom/miniforge/base/lib/python3.9/site-packages/fastapi/routing.py", line 160, in run_endpoint_function
    return await dependant.call(**values)
  File "/Users/beginner/Projects/temp/fastapi-vue/backend/app/apps/permission/views.py", line 98, in getRole
    return respSuccessJson(curd_role.get(db, _id=role_id))
  File "/Users/beginner/Projects/temp/fastapi-vue/backend/app/apps/permission/curd/curd_role.py", line 27, in get
    role = db.query(**queries).filter(self.model.id == _id, self.model.is_deleted == 0).first()
TypeError: sqlalchemy.orm.session.Session.query() argument after ** must be a mapping, not list
WARNING 2022-07-20 15:30:03 StatReload detected file change in 'apps/permission/curd/curd_role.py'. Reloading...
INFO 2022-07-20 15:30:03 Shutting down
INFO 2022-07-20 15:30:03 Waiting for application shutdown.
INFO 2022-07-20 15:30:03 Application shutdown complete.
INFO 2022-07-20 15:30:03 Finished server process [25415]
INFO 2022-07-20 15:30:04 Started server process [25437]
INFO 2022-07-20 15:30:04 Waiting for application startup.
INFO 2022-07-20 15:30:04 Application startup complete.
WARNING 2022-07-20 15:33:52 StatReload detected file change in 'apps/permission/curd/curd_role.py'. Reloading...
INFO 2022-07-20 15:33:52 Shutting down
INFO 2022-07-20 15:33:52 Waiting for application shutdown.
INFO 2022-07-20 15:33:52 Application shutdown complete.
INFO 2022-07-20 15:33:52 Finished server process [25437]
INFO 2022-07-20 15:33:53 Started server process [25485]
INFO 2022-07-20 15:33:53 Waiting for application startup.
INFO 2022-07-20 15:33:53 Application startup complete.
WARNING 2022-07-20 15:34:23 StatReload detected file change in 'apps/permission/curd/curd_role.py'. Reloading...
INFO 2022-07-20 15:34:24 Shutting down
INFO 2022-07-20 15:34:24 Waiting for application shutdown.
INFO 2022-07-20 15:34:24 Application shutdown complete.
INFO 2022-07-20 15:34:24 Finished server process [25485]
INFO 2022-07-20 15:34:24 Started server process [25511]
INFO 2022-07-20 15:34:24 Waiting for application startup.
INFO 2022-07-20 15:34:24 Application startup complete.
WARNING 2022-07-20 15:37:31 StatReload detected file change in 'apps/permission/curd/curd_role.py'. Reloading...
INFO 2022-07-20 15:37:31 Shutting down
INFO 2022-07-20 15:37:31 Waiting for application shutdown.
INFO 2022-07-20 15:37:31 Application shutdown complete.
INFO 2022-07-20 15:37:31 Finished server process [25511]
INFO 2022-07-20 15:37:32 Started server process [25564]
INFO 2022-07-20 15:37:32 Waiting for application startup.
INFO 2022-07-20 15:37:32 Application startup complete.
ERROR 2022-07-20 15:37:44 Exception in ASGI application
Traceback (most recent call last):
  File "/opt/homebrew/Caskroom/miniforge/base/lib/python3.9/site-packages/uvicorn/protocols/http/h11_impl.py", line 366, in run_asgi
    result = await app(self.scope, self.receive, self.send)
  File "/opt/homebrew/Caskroom/miniforge/base/lib/python3.9/site-packages/uvicorn/middleware/proxy_headers.py", line 75, in __call__
    return await self.app(scope, receive, send)
  File "/opt/homebrew/Caskroom/miniforge/base/lib/python3.9/site-packages/uvicorn/middleware/debug.py", line 96, in __call__
    raise exc from None
  File "/opt/homebrew/Caskroom/miniforge/base/lib/python3.9/site-packages/uvicorn/middleware/debug.py", line 93, in __call__
    await self.app(scope, receive, inner_send)
  File "/opt/homebrew/Caskroom/miniforge/base/lib/python3.9/site-packages/fastapi/applications.py", line 261, in __call__
    await super().__call__(scope, receive, send)
  File "/opt/homebrew/Caskroom/miniforge/base/lib/python3.9/site-packages/starlette/applications.py", line 119, in __call__
    await self.middleware_stack(scope, receive, send)
  File "/opt/homebrew/Caskroom/miniforge/base/lib/python3.9/site-packages/starlette/middleware/errors.py", line 181, in __call__
    raise exc
  File "/opt/homebrew/Caskroom/miniforge/base/lib/python3.9/site-packages/starlette/middleware/errors.py", line 159, in __call__
    await self.app(scope, receive, _send)
  File "/opt/homebrew/Caskroom/miniforge/base/lib/python3.9/site-packages/starlette/middleware/cors.py", line 92, in __call__
    await self.simple_response(scope, receive, send, request_headers=headers)
  File "/opt/homebrew/Caskroom/miniforge/base/lib/python3.9/site-packages/starlette/middleware/cors.py", line 147, in simple_response
    await self.app(scope, receive, send)
  File "/opt/homebrew/Caskroom/miniforge/base/lib/python3.9/site-packages/starlette/exceptions.py", line 87, in __call__
    raise exc
  File "/opt/homebrew/Caskroom/miniforge/base/lib/python3.9/site-packages/starlette/exceptions.py", line 76, in __call__
    await self.app(scope, receive, sender)
  File "/opt/homebrew/Caskroom/miniforge/base/lib/python3.9/site-packages/fastapi/middleware/asyncexitstack.py", line 21, in __call__
    raise e
  File "/opt/homebrew/Caskroom/miniforge/base/lib/python3.9/site-packages/fastapi/middleware/asyncexitstack.py", line 18, in __call__
    await self.app(scope, receive, send)
  File "/opt/homebrew/Caskroom/miniforge/base/lib/python3.9/site-packages/starlette/routing.py", line 659, in __call__
    await route.handle(scope, receive, send)
  File "/opt/homebrew/Caskroom/miniforge/base/lib/python3.9/site-packages/starlette/routing.py", line 259, in handle
    await self.app(scope, receive, send)
  File "/opt/homebrew/Caskroom/miniforge/base/lib/python3.9/site-packages/starlette/routing.py", line 61, in app
    response = await func(request)
  File "/opt/homebrew/Caskroom/miniforge/base/lib/python3.9/site-packages/fastapi/routing.py", line 227, in app
    raw_response = await run_endpoint_function(
  File "/opt/homebrew/Caskroom/miniforge/base/lib/python3.9/site-packages/fastapi/routing.py", line 160, in run_endpoint_function
    return await dependant.call(**values)
  File "/Users/beginner/Projects/temp/fastapi-vue/backend/app/apps/permission/views.py", line 98, in getRole
    return respSuccessJson(curd_role.get(db, _id=role_id))
  File "/Users/beginner/Projects/temp/fastapi-vue/backend/app/apps/permission/curd/curd_role.py", line 27, in get
    print(role.dict())
AttributeError: 'NoneType' object has no attribute 'dict'
WARNING 2022-07-20 15:41:05 StatReload detected file change in 'apps/permission/curd/curd_role.py'. Reloading...
INFO 2022-07-20 15:41:05 Shutting down
INFO 2022-07-20 15:41:05 Waiting for application shutdown.
INFO 2022-07-20 15:41:05 Application shutdown complete.
INFO 2022-07-20 15:41:05 Finished server process [25564]
INFO 2022-07-20 15:41:06 Started server process [25636]
INFO 2022-07-20 15:41:06 Waiting for application startup.
INFO 2022-07-20 15:41:06 Application startup complete.
ERROR 2022-07-20 15:41:22 Exception in ASGI application
Traceback (most recent call last):
  File "/opt/homebrew/Caskroom/miniforge/base/lib/python3.9/site-packages/uvicorn/protocols/http/h11_impl.py", line 366, in run_asgi
    result = await app(self.scope, self.receive, self.send)
  File "/opt/homebrew/Caskroom/miniforge/base/lib/python3.9/site-packages/uvicorn/middleware/proxy_headers.py", line 75, in __call__
    return await self.app(scope, receive, send)
  File "/opt/homebrew/Caskroom/miniforge/base/lib/python3.9/site-packages/uvicorn/middleware/debug.py", line 96, in __call__
    raise exc from None
  File "/opt/homebrew/Caskroom/miniforge/base/lib/python3.9/site-packages/uvicorn/middleware/debug.py", line 93, in __call__
    await self.app(scope, receive, inner_send)
  File "/opt/homebrew/Caskroom/miniforge/base/lib/python3.9/site-packages/fastapi/applications.py", line 261, in __call__
    await super().__call__(scope, receive, send)
  File "/opt/homebrew/Caskroom/miniforge/base/lib/python3.9/site-packages/starlette/applications.py", line 119, in __call__
    await self.middleware_stack(scope, receive, send)
  File "/opt/homebrew/Caskroom/miniforge/base/lib/python3.9/site-packages/starlette/middleware/errors.py", line 181, in __call__
    raise exc
  File "/opt/homebrew/Caskroom/miniforge/base/lib/python3.9/site-packages/starlette/middleware/errors.py", line 159, in __call__
    await self.app(scope, receive, _send)
  File "/opt/homebrew/Caskroom/miniforge/base/lib/python3.9/site-packages/starlette/middleware/cors.py", line 92, in __call__
    await self.simple_response(scope, receive, send, request_headers=headers)
  File "/opt/homebrew/Caskroom/miniforge/base/lib/python3.9/site-packages/starlette/middleware/cors.py", line 147, in simple_response
    await self.app(scope, receive, send)
  File "/opt/homebrew/Caskroom/miniforge/base/lib/python3.9/site-packages/starlette/exceptions.py", line 87, in __call__
    raise exc
  File "/opt/homebrew/Caskroom/miniforge/base/lib/python3.9/site-packages/starlette/exceptions.py", line 76, in __call__
    await self.app(scope, receive, sender)
  File "/opt/homebrew/Caskroom/miniforge/base/lib/python3.9/site-packages/fastapi/middleware/asyncexitstack.py", line 21, in __call__
    raise e
  File "/opt/homebrew/Caskroom/miniforge/base/lib/python3.9/site-packages/fastapi/middleware/asyncexitstack.py", line 18, in __call__
    await self.app(scope, receive, send)
  File "/opt/homebrew/Caskroom/miniforge/base/lib/python3.9/site-packages/starlette/routing.py", line 659, in __call__
    await route.handle(scope, receive, send)
  File "/opt/homebrew/Caskroom/miniforge/base/lib/python3.9/site-packages/starlette/routing.py", line 259, in handle
    await self.app(scope, receive, send)
  File "/opt/homebrew/Caskroom/miniforge/base/lib/python3.9/site-packages/starlette/routing.py", line 61, in app
    response = await func(request)
  File "/opt/homebrew/Caskroom/miniforge/base/lib/python3.9/site-packages/fastapi/routing.py", line 227, in app
    raw_response = await run_endpoint_function(
  File "/opt/homebrew/Caskroom/miniforge/base/lib/python3.9/site-packages/fastapi/routing.py", line 160, in run_endpoint_function
    return await dependant.call(**values)
  File "/Users/beginner/Projects/temp/fastapi-vue/backend/app/apps/permission/views.py", line 98, in getRole
    return respSuccessJson(curd_role.get(db, _id=role_id))
  File "/Users/beginner/Projects/temp/fastapi-vue/backend/app/apps/permission/curd/curd_role.py", line 27, in get
    print(role.dict())
AttributeError: Could not locate column in row for column 'dict'
WARNING 2022-07-20 15:41:44 StatReload detected file change in 'apps/permission/curd/curd_role.py'. Reloading...
INFO 2022-07-20 15:41:44 Shutting down
INFO 2022-07-20 15:41:44 Waiting for application shutdown.
INFO 2022-07-20 15:41:44 Application shutdown complete.
INFO 2022-07-20 15:41:44 Finished server process [25636]
INFO 2022-07-20 15:41:45 Started server process [25648]
INFO 2022-07-20 15:41:45 Waiting for application startup.
INFO 2022-07-20 15:41:45 Application startup complete.
WARNING 2022-07-20 15:42:34 StatReload detected file change in 'apps/permission/curd/curd_role.py'. Reloading...
INFO 2022-07-20 15:42:34 Shutting down
INFO 2022-07-20 15:42:34 Waiting for application shutdown.
INFO 2022-07-20 15:42:34 Application shutdown complete.
INFO 2022-07-20 15:42:34 Finished server process [25648]
INFO 2022-07-20 15:42:34 Started server process [25669]
INFO 2022-07-20 15:42:34 Waiting for application startup.
INFO 2022-07-20 15:42:34 Application startup complete.
WARNING 2022-07-20 15:44:27 StatReload detected file change in 'apps/permission/curd/curd_role.py'. Reloading...
INFO 2022-07-20 15:44:27 Shutting down
INFO 2022-07-20 15:44:27 Waiting for application shutdown.
INFO 2022-07-20 15:44:27 Application shutdown complete.
INFO 2022-07-20 15:44:27 Finished server process [25669]
INFO 2022-07-20 15:44:28 Started server process [25704]
INFO 2022-07-20 15:44:28 Waiting for application startup.
INFO 2022-07-20 15:44:28 Application startup complete.
ERROR 2022-07-20 15:44:36 Exception in ASGI application
Traceback (most recent call last):
  File "/opt/homebrew/Caskroom/miniforge/base/lib/python3.9/site-packages/uvicorn/protocols/http/h11_impl.py", line 366, in run_asgi
    result = await app(self.scope, self.receive, self.send)
  File "/opt/homebrew/Caskroom/miniforge/base/lib/python3.9/site-packages/uvicorn/middleware/proxy_headers.py", line 75, in __call__
    return await self.app(scope, receive, send)
  File "/opt/homebrew/Caskroom/miniforge/base/lib/python3.9/site-packages/uvicorn/middleware/debug.py", line 96, in __call__
    raise exc from None
  File "/opt/homebrew/Caskroom/miniforge/base/lib/python3.9/site-packages/uvicorn/middleware/debug.py", line 93, in __call__
    await self.app(scope, receive, inner_send)
  File "/opt/homebrew/Caskroom/miniforge/base/lib/python3.9/site-packages/fastapi/applications.py", line 261, in __call__
    await super().__call__(scope, receive, send)
  File "/opt/homebrew/Caskroom/miniforge/base/lib/python3.9/site-packages/starlette/applications.py", line 119, in __call__
    await self.middleware_stack(scope, receive, send)
  File "/opt/homebrew/Caskroom/miniforge/base/lib/python3.9/site-packages/starlette/middleware/errors.py", line 181, in __call__
    raise exc
  File "/opt/homebrew/Caskroom/miniforge/base/lib/python3.9/site-packages/starlette/middleware/errors.py", line 159, in __call__
    await self.app(scope, receive, _send)
  File "/opt/homebrew/Caskroom/miniforge/base/lib/python3.9/site-packages/starlette/middleware/cors.py", line 92, in __call__
    await self.simple_response(scope, receive, send, request_headers=headers)
  File "/opt/homebrew/Caskroom/miniforge/base/lib/python3.9/site-packages/starlette/middleware/cors.py", line 147, in simple_response
    await self.app(scope, receive, send)
  File "/opt/homebrew/Caskroom/miniforge/base/lib/python3.9/site-packages/starlette/exceptions.py", line 87, in __call__
    raise exc
  File "/opt/homebrew/Caskroom/miniforge/base/lib/python3.9/site-packages/starlette/exceptions.py", line 76, in __call__
    await self.app(scope, receive, sender)
  File "/opt/homebrew/Caskroom/miniforge/base/lib/python3.9/site-packages/fastapi/middleware/asyncexitstack.py", line 21, in __call__
    raise e
  File "/opt/homebrew/Caskroom/miniforge/base/lib/python3.9/site-packages/fastapi/middleware/asyncexitstack.py", line 18, in __call__
    await self.app(scope, receive, send)
  File "/opt/homebrew/Caskroom/miniforge/base/lib/python3.9/site-packages/starlette/routing.py", line 659, in __call__
    await route.handle(scope, receive, send)
  File "/opt/homebrew/Caskroom/miniforge/base/lib/python3.9/site-packages/starlette/routing.py", line 259, in handle
    await self.app(scope, receive, send)
  File "/opt/homebrew/Caskroom/miniforge/base/lib/python3.9/site-packages/starlette/routing.py", line 61, in app
    response = await func(request)
  File "/opt/homebrew/Caskroom/miniforge/base/lib/python3.9/site-packages/fastapi/routing.py", line 227, in app
    raw_response = await run_endpoint_function(
  File "/opt/homebrew/Caskroom/miniforge/base/lib/python3.9/site-packages/fastapi/routing.py", line 160, in run_endpoint_function
    return await dependant.call(**values)
  File "/Users/beginner/Projects/temp/fastapi-vue/backend/app/apps/permission/views.py", line 98, in getRole
    return respSuccessJson(curd_role.get(db, _id=role_id))
  File "/Users/beginner/Projects/temp/fastapi-vue/backend/app/apps/permission/curd/curd_role.py", line 27, in get
    print(role.role_menu)
AttributeError: 'list' object has no attribute 'role_menu'
WARNING 2022-07-20 15:44:48 StatReload detected file change in 'apps/permission/curd/curd_role.py'. Reloading...
INFO 2022-07-20 15:44:48 Shutting down
INFO 2022-07-20 15:44:48 Waiting for application shutdown.
INFO 2022-07-20 15:44:48 Application shutdown complete.
INFO 2022-07-20 15:44:48 Finished server process [25704]
INFO 2022-07-20 15:44:48 Started server process [25713]
INFO 2022-07-20 15:44:48 Waiting for application startup.
INFO 2022-07-20 15:44:48 Application startup complete.
WARNING 2022-07-20 15:45:57 StatReload detected file change in 'apps/permission/curd/curd_role.py'. Reloading...
INFO 2022-07-20 15:45:58 Shutting down
INFO 2022-07-20 15:45:58 Waiting for application shutdown.
INFO 2022-07-20 15:45:58 Application shutdown complete.
INFO 2022-07-20 15:45:58 Finished server process [25713]
INFO 2022-07-20 15:45:58 Started server process [25734]
INFO 2022-07-20 15:45:58 Waiting for application startup.
INFO 2022-07-20 15:45:58 Application startup complete.
WARNING 2022-07-20 15:47:25 StatReload detected file change in 'apps/permission/curd/curd_role.py'. Reloading...
INFO 2022-07-20 15:47:25 Shutting down
INFO 2022-07-20 15:47:26 Waiting for application shutdown.
INFO 2022-07-20 15:47:26 Application shutdown complete.
INFO 2022-07-20 15:47:26 Finished server process [25734]
INFO 2022-07-20 15:47:26 Started server process [25766]
INFO 2022-07-20 15:47:26 Waiting for application startup.
INFO 2022-07-20 15:47:26 Application startup complete.
WARNING 2022-07-20 15:48:05 StatReload detected file change in 'apps/permission/curd/curd_role.py'. Reloading...
INFO 2022-07-20 15:48:05 Shutting down
INFO 2022-07-20 15:48:05 Waiting for application shutdown.
INFO 2022-07-20 15:48:05 Application shutdown complete.
INFO 2022-07-20 15:48:05 Finished server process [25766]
INFO 2022-07-20 15:48:06 Started server process [25795]
INFO 2022-07-20 15:48:06 Waiting for application startup.
INFO 2022-07-20 15:48:06 Application startup complete.
ERROR 2022-07-20 15:48:13 Exception in ASGI application
Traceback (most recent call last):
  File "/opt/homebrew/Caskroom/miniforge/base/lib/python3.9/site-packages/uvicorn/protocols/http/h11_impl.py", line 366, in run_asgi
    result = await app(self.scope, self.receive, self.send)
  File "/opt/homebrew/Caskroom/miniforge/base/lib/python3.9/site-packages/uvicorn/middleware/proxy_headers.py", line 75, in __call__
    return await self.app(scope, receive, send)
  File "/opt/homebrew/Caskroom/miniforge/base/lib/python3.9/site-packages/uvicorn/middleware/debug.py", line 96, in __call__
    raise exc from None
  File "/opt/homebrew/Caskroom/miniforge/base/lib/python3.9/site-packages/uvicorn/middleware/debug.py", line 93, in __call__
    await self.app(scope, receive, inner_send)
  File "/opt/homebrew/Caskroom/miniforge/base/lib/python3.9/site-packages/fastapi/applications.py", line 261, in __call__
    await super().__call__(scope, receive, send)
  File "/opt/homebrew/Caskroom/miniforge/base/lib/python3.9/site-packages/starlette/applications.py", line 119, in __call__
    await self.middleware_stack(scope, receive, send)
  File "/opt/homebrew/Caskroom/miniforge/base/lib/python3.9/site-packages/starlette/middleware/errors.py", line 181, in __call__
    raise exc
  File "/opt/homebrew/Caskroom/miniforge/base/lib/python3.9/site-packages/starlette/middleware/errors.py", line 159, in __call__
    await self.app(scope, receive, _send)
  File "/opt/homebrew/Caskroom/miniforge/base/lib/python3.9/site-packages/starlette/middleware/cors.py", line 92, in __call__
    await self.simple_response(scope, receive, send, request_headers=headers)
  File "/opt/homebrew/Caskroom/miniforge/base/lib/python3.9/site-packages/starlette/middleware/cors.py", line 147, in simple_response
    await self.app(scope, receive, send)
  File "/opt/homebrew/Caskroom/miniforge/base/lib/python3.9/site-packages/starlette/exceptions.py", line 87, in __call__
    raise exc
  File "/opt/homebrew/Caskroom/miniforge/base/lib/python3.9/site-packages/starlette/exceptions.py", line 76, in __call__
    await self.app(scope, receive, sender)
  File "/opt/homebrew/Caskroom/miniforge/base/lib/python3.9/site-packages/fastapi/middleware/asyncexitstack.py", line 21, in __call__
    raise e
  File "/opt/homebrew/Caskroom/miniforge/base/lib/python3.9/site-packages/fastapi/middleware/asyncexitstack.py", line 18, in __call__
    await self.app(scope, receive, send)
  File "/opt/homebrew/Caskroom/miniforge/base/lib/python3.9/site-packages/starlette/routing.py", line 659, in __call__
    await route.handle(scope, receive, send)
  File "/opt/homebrew/Caskroom/miniforge/base/lib/python3.9/site-packages/starlette/routing.py", line 259, in handle
    await self.app(scope, receive, send)
  File "/opt/homebrew/Caskroom/miniforge/base/lib/python3.9/site-packages/starlette/routing.py", line 61, in app
    response = await func(request)
  File "/opt/homebrew/Caskroom/miniforge/base/lib/python3.9/site-packages/fastapi/routing.py", line 227, in app
    raw_response = await run_endpoint_function(
  File "/opt/homebrew/Caskroom/miniforge/base/lib/python3.9/site-packages/fastapi/routing.py", line 160, in run_endpoint_function
    return await dependant.call(**values)
  File "/Users/beginner/Projects/temp/fastapi-vue/backend/app/apps/permission/views.py", line 98, in getRole
    return respSuccessJson(curd_role.get(db, _id=role_id))
  File "/Users/beginner/Projects/temp/fastapi-vue/backend/app/apps/permission/curd/curd_role.py", line 27, in get
    print([i.all() for i in role.role_menu])
  File "/Users/beginner/Projects/temp/fastapi-vue/backend/app/apps/permission/curd/curd_role.py", line 27, in <listcomp>
    print([i.all() for i in role.role_menu])
AttributeError: 'Menus' object has no attribute 'all'
WARNING 2022-07-20 15:48:42 StatReload detected file change in 'apps/permission/curd/curd_role.py'. Reloading...
INFO 2022-07-20 15:48:42 Shutting down
INFO 2022-07-20 15:48:42 Waiting for application shutdown.
INFO 2022-07-20 15:48:42 Application shutdown complete.
INFO 2022-07-20 15:48:42 Finished server process [25795]
INFO 2022-07-20 15:48:43 Started server process [25806]
INFO 2022-07-20 15:48:43 Waiting for application startup.
INFO 2022-07-20 15:48:43 Application startup complete.
ERROR 2022-07-20 15:48:49 Exception in ASGI application
Traceback (most recent call last):
  File "/opt/homebrew/Caskroom/miniforge/base/lib/python3.9/site-packages/uvicorn/protocols/http/h11_impl.py", line 366, in run_asgi
    result = await app(self.scope, self.receive, self.send)
  File "/opt/homebrew/Caskroom/miniforge/base/lib/python3.9/site-packages/uvicorn/middleware/proxy_headers.py", line 75, in __call__
    return await self.app(scope, receive, send)
  File "/opt/homebrew/Caskroom/miniforge/base/lib/python3.9/site-packages/uvicorn/middleware/debug.py", line 96, in __call__
    raise exc from None
  File "/opt/homebrew/Caskroom/miniforge/base/lib/python3.9/site-packages/uvicorn/middleware/debug.py", line 93, in __call__
    await self.app(scope, receive, inner_send)
  File "/opt/homebrew/Caskroom/miniforge/base/lib/python3.9/site-packages/fastapi/applications.py", line 261, in __call__
    await super().__call__(scope, receive, send)
  File "/opt/homebrew/Caskroom/miniforge/base/lib/python3.9/site-packages/starlette/applications.py", line 119, in __call__
    await self.middleware_stack(scope, receive, send)
  File "/opt/homebrew/Caskroom/miniforge/base/lib/python3.9/site-packages/starlette/middleware/errors.py", line 181, in __call__
    raise exc
  File "/opt/homebrew/Caskroom/miniforge/base/lib/python3.9/site-packages/starlette/middleware/errors.py", line 159, in __call__
    await self.app(scope, receive, _send)
  File "/opt/homebrew/Caskroom/miniforge/base/lib/python3.9/site-packages/starlette/middleware/cors.py", line 92, in __call__
    await self.simple_response(scope, receive, send, request_headers=headers)
  File "/opt/homebrew/Caskroom/miniforge/base/lib/python3.9/site-packages/starlette/middleware/cors.py", line 147, in simple_response
    await self.app(scope, receive, send)
  File "/opt/homebrew/Caskroom/miniforge/base/lib/python3.9/site-packages/starlette/exceptions.py", line 87, in __call__
    raise exc
  File "/opt/homebrew/Caskroom/miniforge/base/lib/python3.9/site-packages/starlette/exceptions.py", line 76, in __call__
    await self.app(scope, receive, sender)
  File "/opt/homebrew/Caskroom/miniforge/base/lib/python3.9/site-packages/fastapi/middleware/asyncexitstack.py", line 21, in __call__
    raise e
  File "/opt/homebrew/Caskroom/miniforge/base/lib/python3.9/site-packages/fastapi/middleware/asyncexitstack.py", line 18, in __call__
    await self.app(scope, receive, send)
  File "/opt/homebrew/Caskroom/miniforge/base/lib/python3.9/site-packages/starlette/routing.py", line 659, in __call__
    await route.handle(scope, receive, send)
  File "/opt/homebrew/Caskroom/miniforge/base/lib/python3.9/site-packages/starlette/routing.py", line 259, in handle
    await self.app(scope, receive, send)
  File "/opt/homebrew/Caskroom/miniforge/base/lib/python3.9/site-packages/starlette/routing.py", line 61, in app
    response = await func(request)
  File "/opt/homebrew/Caskroom/miniforge/base/lib/python3.9/site-packages/fastapi/routing.py", line 227, in app
    raw_response = await run_endpoint_function(
  File "/opt/homebrew/Caskroom/miniforge/base/lib/python3.9/site-packages/fastapi/routing.py", line 160, in run_endpoint_function
    return await dependant.call(**values)
  File "/Users/beginner/Projects/temp/fastapi-vue/backend/app/apps/permission/views.py", line 98, in getRole
    return respSuccessJson(curd_role.get(db, _id=role_id))
  File "/Users/beginner/Projects/temp/fastapi-vue/backend/app/apps/permission/curd/curd_role.py", line 27, in get
    print([db.query(i).filter() for i in role.role_menu])
  File "/Users/beginner/Projects/temp/fastapi-vue/backend/app/apps/permission/curd/curd_role.py", line 27, in <listcomp>
    print([db.query(i).filter() for i in role.role_menu])
  File "/opt/homebrew/Caskroom/miniforge/base/lib/python3.9/site-packages/sqlalchemy/orm/session.py", line 2143, in query
    return self._query_cls(entities, self, **kwargs)
  File "/opt/homebrew/Caskroom/miniforge/base/lib/python3.9/site-packages/sqlalchemy/orm/query.py", line 180, in __init__
    self._set_entities(entities)
  File "/opt/homebrew/Caskroom/miniforge/base/lib/python3.9/site-packages/sqlalchemy/orm/query.py", line 187, in _set_entities
    self._raw_columns = [
  File "/opt/homebrew/Caskroom/miniforge/base/lib/python3.9/site-packages/sqlalchemy/orm/query.py", line 188, in <listcomp>
    coercions.expect(
  File "/opt/homebrew/Caskroom/miniforge/base/lib/python3.9/site-packages/sqlalchemy/sql/coercions.py", line 181, in expect
    insp._post_inspect
AttributeError: 'InstanceState' object has no attribute '_post_inspect'
WARNING 2022-07-20 15:49:00 StatReload detected file change in 'apps/permission/curd/curd_role.py'. Reloading...
INFO 2022-07-20 15:49:00 Shutting down
INFO 2022-07-20 15:49:00 Waiting for application shutdown.
INFO 2022-07-20 15:49:00 Application shutdown complete.
INFO 2022-07-20 15:49:00 Finished server process [25806]
INFO 2022-07-20 15:49:01 Started server process [25819]
INFO 2022-07-20 15:49:01 Waiting for application startup.
INFO 2022-07-20 15:49:01 Application startup complete.
ERROR 2022-07-20 15:49:04 Exception in ASGI application
Traceback (most recent call last):
  File "/opt/homebrew/Caskroom/miniforge/base/lib/python3.9/site-packages/uvicorn/protocols/http/h11_impl.py", line 366, in run_asgi
    result = await app(self.scope, self.receive, self.send)
  File "/opt/homebrew/Caskroom/miniforge/base/lib/python3.9/site-packages/uvicorn/middleware/proxy_headers.py", line 75, in __call__
    return await self.app(scope, receive, send)
  File "/opt/homebrew/Caskroom/miniforge/base/lib/python3.9/site-packages/uvicorn/middleware/debug.py", line 96, in __call__
    raise exc from None
  File "/opt/homebrew/Caskroom/miniforge/base/lib/python3.9/site-packages/uvicorn/middleware/debug.py", line 93, in __call__
    await self.app(scope, receive, inner_send)
  File "/opt/homebrew/Caskroom/miniforge/base/lib/python3.9/site-packages/fastapi/applications.py", line 261, in __call__
    await super().__call__(scope, receive, send)
  File "/opt/homebrew/Caskroom/miniforge/base/lib/python3.9/site-packages/starlette/applications.py", line 119, in __call__
    await self.middleware_stack(scope, receive, send)
  File "/opt/homebrew/Caskroom/miniforge/base/lib/python3.9/site-packages/starlette/middleware/errors.py", line 181, in __call__
    raise exc
  File "/opt/homebrew/Caskroom/miniforge/base/lib/python3.9/site-packages/starlette/middleware/errors.py", line 159, in __call__
    await self.app(scope, receive, _send)
  File "/opt/homebrew/Caskroom/miniforge/base/lib/python3.9/site-packages/starlette/middleware/cors.py", line 92, in __call__
    await self.simple_response(scope, receive, send, request_headers=headers)
  File "/opt/homebrew/Caskroom/miniforge/base/lib/python3.9/site-packages/starlette/middleware/cors.py", line 147, in simple_response
    await self.app(scope, receive, send)
  File "/opt/homebrew/Caskroom/miniforge/base/lib/python3.9/site-packages/starlette/exceptions.py", line 87, in __call__
    raise exc
  File "/opt/homebrew/Caskroom/miniforge/base/lib/python3.9/site-packages/starlette/exceptions.py", line 76, in __call__
    await self.app(scope, receive, sender)
  File "/opt/homebrew/Caskroom/miniforge/base/lib/python3.9/site-packages/fastapi/middleware/asyncexitstack.py", line 21, in __call__
    raise e
  File "/opt/homebrew/Caskroom/miniforge/base/lib/python3.9/site-packages/fastapi/middleware/asyncexitstack.py", line 18, in __call__
    await self.app(scope, receive, send)
  File "/opt/homebrew/Caskroom/miniforge/base/lib/python3.9/site-packages/starlette/routing.py", line 659, in __call__
    await route.handle(scope, receive, send)
  File "/opt/homebrew/Caskroom/miniforge/base/lib/python3.9/site-packages/starlette/routing.py", line 259, in handle
    await self.app(scope, receive, send)
  File "/opt/homebrew/Caskroom/miniforge/base/lib/python3.9/site-packages/starlette/routing.py", line 61, in app
    response = await func(request)
  File "/opt/homebrew/Caskroom/miniforge/base/lib/python3.9/site-packages/fastapi/routing.py", line 227, in app
    raw_response = await run_endpoint_function(
  File "/opt/homebrew/Caskroom/miniforge/base/lib/python3.9/site-packages/fastapi/routing.py", line 160, in run_endpoint_function
    return await dependant.call(**values)
  File "/Users/beginner/Projects/temp/fastapi-vue/backend/app/apps/permission/views.py", line 98, in getRole
    return respSuccessJson(curd_role.get(db, _id=role_id))
  File "/Users/beginner/Projects/temp/fastapi-vue/backend/app/apps/permission/curd/curd_role.py", line 27, in get
    print([db.query(i).first() for i in role.role_menu])
  File "/Users/beginner/Projects/temp/fastapi-vue/backend/app/apps/permission/curd/curd_role.py", line 27, in <listcomp>
    print([db.query(i).first() for i in role.role_menu])
  File "/opt/homebrew/Caskroom/miniforge/base/lib/python3.9/site-packages/sqlalchemy/orm/session.py", line 2143, in query
    return self._query_cls(entities, self, **kwargs)
  File "/opt/homebrew/Caskroom/miniforge/base/lib/python3.9/site-packages/sqlalchemy/orm/query.py", line 180, in __init__
    self._set_entities(entities)
  File "/opt/homebrew/Caskroom/miniforge/base/lib/python3.9/site-packages/sqlalchemy/orm/query.py", line 187, in _set_entities
    self._raw_columns = [
  File "/opt/homebrew/Caskroom/miniforge/base/lib/python3.9/site-packages/sqlalchemy/orm/query.py", line 188, in <listcomp>
    coercions.expect(
  File "/opt/homebrew/Caskroom/miniforge/base/lib/python3.9/site-packages/sqlalchemy/sql/coercions.py", line 181, in expect
    insp._post_inspect
AttributeError: 'InstanceState' object has no attribute '_post_inspect'
WARNING 2022-07-20 15:49:33 StatReload detected file change in 'apps/permission/curd/curd_role.py'. Reloading...
INFO 2022-07-20 15:49:33 Shutting down
INFO 2022-07-20 15:49:33 Waiting for application shutdown.
INFO 2022-07-20 15:49:33 Application shutdown complete.
INFO 2022-07-20 15:49:33 Finished server process [25819]
INFO 2022-07-20 15:49:34 Started server process [25829]
INFO 2022-07-20 15:49:34 Waiting for application startup.
INFO 2022-07-20 15:49:34 Application startup complete.
ERROR 2022-07-20 15:49:39 Exception in ASGI application
Traceback (most recent call last):
  File "/opt/homebrew/Caskroom/miniforge/base/lib/python3.9/site-packages/uvicorn/protocols/http/h11_impl.py", line 366, in run_asgi
    result = await app(self.scope, self.receive, self.send)
  File "/opt/homebrew/Caskroom/miniforge/base/lib/python3.9/site-packages/uvicorn/middleware/proxy_headers.py", line 75, in __call__
    return await self.app(scope, receive, send)
  File "/opt/homebrew/Caskroom/miniforge/base/lib/python3.9/site-packages/uvicorn/middleware/debug.py", line 96, in __call__
    raise exc from None
  File "/opt/homebrew/Caskroom/miniforge/base/lib/python3.9/site-packages/uvicorn/middleware/debug.py", line 93, in __call__
    await self.app(scope, receive, inner_send)
  File "/opt/homebrew/Caskroom/miniforge/base/lib/python3.9/site-packages/fastapi/applications.py", line 261, in __call__
    await super().__call__(scope, receive, send)
  File "/opt/homebrew/Caskroom/miniforge/base/lib/python3.9/site-packages/starlette/applications.py", line 119, in __call__
    await self.middleware_stack(scope, receive, send)
  File "/opt/homebrew/Caskroom/miniforge/base/lib/python3.9/site-packages/starlette/middleware/errors.py", line 181, in __call__
    raise exc
  File "/opt/homebrew/Caskroom/miniforge/base/lib/python3.9/site-packages/starlette/middleware/errors.py", line 159, in __call__
    await self.app(scope, receive, _send)
  File "/opt/homebrew/Caskroom/miniforge/base/lib/python3.9/site-packages/starlette/middleware/cors.py", line 92, in __call__
    await self.simple_response(scope, receive, send, request_headers=headers)
  File "/opt/homebrew/Caskroom/miniforge/base/lib/python3.9/site-packages/starlette/middleware/cors.py", line 147, in simple_response
    await self.app(scope, receive, send)
  File "/opt/homebrew/Caskroom/miniforge/base/lib/python3.9/site-packages/starlette/exceptions.py", line 87, in __call__
    raise exc
  File "/opt/homebrew/Caskroom/miniforge/base/lib/python3.9/site-packages/starlette/exceptions.py", line 76, in __call__
    await self.app(scope, receive, sender)
  File "/opt/homebrew/Caskroom/miniforge/base/lib/python3.9/site-packages/fastapi/middleware/asyncexitstack.py", line 21, in __call__
    raise e
  File "/opt/homebrew/Caskroom/miniforge/base/lib/python3.9/site-packages/fastapi/middleware/asyncexitstack.py", line 18, in __call__
    await self.app(scope, receive, send)
  File "/opt/homebrew/Caskroom/miniforge/base/lib/python3.9/site-packages/starlette/routing.py", line 659, in __call__
    await route.handle(scope, receive, send)
  File "/opt/homebrew/Caskroom/miniforge/base/lib/python3.9/site-packages/starlette/routing.py", line 259, in handle
    await self.app(scope, receive, send)
  File "/opt/homebrew/Caskroom/miniforge/base/lib/python3.9/site-packages/starlette/routing.py", line 61, in app
    response = await func(request)
  File "/opt/homebrew/Caskroom/miniforge/base/lib/python3.9/site-packages/fastapi/routing.py", line 227, in app
    raw_response = await run_endpoint_function(
  File "/opt/homebrew/Caskroom/miniforge/base/lib/python3.9/site-packages/fastapi/routing.py", line 160, in run_endpoint_function
    return await dependant.call(**values)
  File "/Users/beginner/Projects/temp/fastapi-vue/backend/app/apps/permission/views.py", line 98, in getRole
    return respSuccessJson(curd_role.get(db, _id=role_id))
  File "/Users/beginner/Projects/temp/fastapi-vue/backend/app/apps/permission/curd/curd_role.py", line 27, in get
    print([i['id'] for i in role.role_menu])
  File "/Users/beginner/Projects/temp/fastapi-vue/backend/app/apps/permission/curd/curd_role.py", line 27, in <listcomp>
    print([i['id'] for i in role.role_menu])
TypeError: 'Menus' object is not subscriptable
WARNING 2022-07-20 15:49:50 StatReload detected file change in 'apps/permission/curd/curd_role.py'. Reloading...
INFO 2022-07-20 15:49:50 Shutting down
INFO 2022-07-20 15:49:50 Waiting for application shutdown.
INFO 2022-07-20 15:49:50 Application shutdown complete.
INFO 2022-07-20 15:49:50 Finished server process [25829]
INFO 2022-07-20 15:49:51 Started server process [25833]
INFO 2022-07-20 15:49:51 Waiting for application startup.
INFO 2022-07-20 15:49:51 Application startup complete.
WARNING 2022-07-20 15:51:58 StatReload detected file change in 'apps/permission/curd/curd_role.py'. Reloading...
INFO 2022-07-20 15:51:58 Shutting down
INFO 2022-07-20 15:51:58 Waiting for application shutdown.
INFO 2022-07-20 15:51:58 Application shutdown complete.
INFO 2022-07-20 15:51:58 Finished server process [25833]
INFO 2022-07-20 15:51:59 Started server process [25857]
INFO 2022-07-20 15:51:59 Waiting for application startup.
INFO 2022-07-20 15:51:59 Application startup complete.
WARNING 2022-07-20 15:53:43 StatReload detected file change in 'apps/permission/curd/curd_role.py'. Reloading...
INFO 2022-07-20 15:53:43 Shutting down
INFO 2022-07-20 15:53:43 Waiting for application shutdown.
INFO 2022-07-20 15:53:43 Application shutdown complete.
INFO 2022-07-20 15:53:43 Finished server process [25857]
INFO 2022-07-20 15:53:44 Started server process [25900]
INFO 2022-07-20 15:53:44 Waiting for application startup.
INFO 2022-07-20 15:53:44 Application startup complete.
WARNING 2022-07-20 15:56:42 StatReload detected file change in 'apps/permission/curd/curd_role.py'. Reloading...
INFO 2022-07-20 15:56:42 Shutting down
INFO 2022-07-20 15:56:42 Waiting for application shutdown.
INFO 2022-07-20 15:56:42 Application shutdown complete.
INFO 2022-07-20 15:56:42 Finished server process [25900]
INFO 2022-07-20 15:56:43 Started server process [25963]
INFO 2022-07-20 15:56:43 Waiting for application startup.
INFO 2022-07-20 15:56:43 Application startup complete.
ERROR 2022-07-20 15:56:47 Exception in ASGI application
Traceback (most recent call last):
  File "/opt/homebrew/Caskroom/miniforge/base/lib/python3.9/site-packages/uvicorn/protocols/http/h11_impl.py", line 366, in run_asgi
    result = await app(self.scope, self.receive, self.send)
  File "/opt/homebrew/Caskroom/miniforge/base/lib/python3.9/site-packages/uvicorn/middleware/proxy_headers.py", line 75, in __call__
    return await self.app(scope, receive, send)
  File "/opt/homebrew/Caskroom/miniforge/base/lib/python3.9/site-packages/uvicorn/middleware/debug.py", line 96, in __call__
    raise exc from None
  File "/opt/homebrew/Caskroom/miniforge/base/lib/python3.9/site-packages/uvicorn/middleware/debug.py", line 93, in __call__
    await self.app(scope, receive, inner_send)
  File "/opt/homebrew/Caskroom/miniforge/base/lib/python3.9/site-packages/fastapi/applications.py", line 261, in __call__
    await super().__call__(scope, receive, send)
  File "/opt/homebrew/Caskroom/miniforge/base/lib/python3.9/site-packages/starlette/applications.py", line 119, in __call__
    await self.middleware_stack(scope, receive, send)
  File "/opt/homebrew/Caskroom/miniforge/base/lib/python3.9/site-packages/starlette/middleware/errors.py", line 181, in __call__
    raise exc
  File "/opt/homebrew/Caskroom/miniforge/base/lib/python3.9/site-packages/starlette/middleware/errors.py", line 159, in __call__
    await self.app(scope, receive, _send)
  File "/opt/homebrew/Caskroom/miniforge/base/lib/python3.9/site-packages/starlette/middleware/cors.py", line 92, in __call__
    await self.simple_response(scope, receive, send, request_headers=headers)
  File "/opt/homebrew/Caskroom/miniforge/base/lib/python3.9/site-packages/starlette/middleware/cors.py", line 147, in simple_response
    await self.app(scope, receive, send)
  File "/opt/homebrew/Caskroom/miniforge/base/lib/python3.9/site-packages/starlette/exceptions.py", line 87, in __call__
    raise exc
  File "/opt/homebrew/Caskroom/miniforge/base/lib/python3.9/site-packages/starlette/exceptions.py", line 76, in __call__
    await self.app(scope, receive, sender)
  File "/opt/homebrew/Caskroom/miniforge/base/lib/python3.9/site-packages/fastapi/middleware/asyncexitstack.py", line 21, in __call__
    raise e
  File "/opt/homebrew/Caskroom/miniforge/base/lib/python3.9/site-packages/fastapi/middleware/asyncexitstack.py", line 18, in __call__
    await self.app(scope, receive, send)
  File "/opt/homebrew/Caskroom/miniforge/base/lib/python3.9/site-packages/starlette/routing.py", line 659, in __call__
    await route.handle(scope, receive, send)
  File "/opt/homebrew/Caskroom/miniforge/base/lib/python3.9/site-packages/starlette/routing.py", line 259, in handle
    await self.app(scope, receive, send)
  File "/opt/homebrew/Caskroom/miniforge/base/lib/python3.9/site-packages/starlette/routing.py", line 61, in app
    response = await func(request)
  File "/opt/homebrew/Caskroom/miniforge/base/lib/python3.9/site-packages/fastapi/routing.py", line 227, in app
    raw_response = await run_endpoint_function(
  File "/opt/homebrew/Caskroom/miniforge/base/lib/python3.9/site-packages/fastapi/routing.py", line 160, in run_endpoint_function
    return await dependant.call(**values)
  File "/Users/beginner/Projects/temp/fastapi-vue/backend/app/apps/permission/views.py", line 98, in getRole
    return respSuccessJson(curd_role.get(db, _id=role_id))
  File "/Users/beginner/Projects/temp/fastapi-vue/backend/app/apps/permission/curd/curd_role.py", line 27, in get
    res = role.dict()
AttributeError: Could not locate column in row for column 'dict'
WARNING 2022-07-20 15:57:07 StatReload detected file change in 'apps/permission/curd/curd_role.py'. Reloading...
INFO 2022-07-20 15:57:07 Shutting down
INFO 2022-07-20 15:57:07 Waiting for application shutdown.
INFO 2022-07-20 15:57:07 Application shutdown complete.
INFO 2022-07-20 15:57:07 Finished server process [25963]
INFO 2022-07-20 15:57:08 Started server process [25988]
INFO 2022-07-20 15:57:08 Waiting for application startup.
INFO 2022-07-20 15:57:08 Application startup complete.
ERROR 2022-07-20 15:57:14 Exception in ASGI application
Traceback (most recent call last):
  File "/opt/homebrew/Caskroom/miniforge/base/lib/python3.9/site-packages/uvicorn/protocols/http/h11_impl.py", line 366, in run_asgi
    result = await app(self.scope, self.receive, self.send)
  File "/opt/homebrew/Caskroom/miniforge/base/lib/python3.9/site-packages/uvicorn/middleware/proxy_headers.py", line 75, in __call__
    return await self.app(scope, receive, send)
  File "/opt/homebrew/Caskroom/miniforge/base/lib/python3.9/site-packages/uvicorn/middleware/debug.py", line 96, in __call__
    raise exc from None
  File "/opt/homebrew/Caskroom/miniforge/base/lib/python3.9/site-packages/uvicorn/middleware/debug.py", line 93, in __call__
    await self.app(scope, receive, inner_send)
  File "/opt/homebrew/Caskroom/miniforge/base/lib/python3.9/site-packages/fastapi/applications.py", line 261, in __call__
    await super().__call__(scope, receive, send)
  File "/opt/homebrew/Caskroom/miniforge/base/lib/python3.9/site-packages/starlette/applications.py", line 119, in __call__
    await self.middleware_stack(scope, receive, send)
  File "/opt/homebrew/Caskroom/miniforge/base/lib/python3.9/site-packages/starlette/middleware/errors.py", line 181, in __call__
    raise exc
  File "/opt/homebrew/Caskroom/miniforge/base/lib/python3.9/site-packages/starlette/middleware/errors.py", line 159, in __call__
    await self.app(scope, receive, _send)
  File "/opt/homebrew/Caskroom/miniforge/base/lib/python3.9/site-packages/starlette/middleware/cors.py", line 92, in __call__
    await self.simple_response(scope, receive, send, request_headers=headers)
  File "/opt/homebrew/Caskroom/miniforge/base/lib/python3.9/site-packages/starlette/middleware/cors.py", line 147, in simple_response
    await self.app(scope, receive, send)
  File "/opt/homebrew/Caskroom/miniforge/base/lib/python3.9/site-packages/starlette/exceptions.py", line 87, in __call__
    raise exc
  File "/opt/homebrew/Caskroom/miniforge/base/lib/python3.9/site-packages/starlette/exceptions.py", line 76, in __call__
    await self.app(scope, receive, sender)
  File "/opt/homebrew/Caskroom/miniforge/base/lib/python3.9/site-packages/fastapi/middleware/asyncexitstack.py", line 21, in __call__
    raise e
  File "/opt/homebrew/Caskroom/miniforge/base/lib/python3.9/site-packages/fastapi/middleware/asyncexitstack.py", line 18, in __call__
    await self.app(scope, receive, send)
  File "/opt/homebrew/Caskroom/miniforge/base/lib/python3.9/site-packages/starlette/routing.py", line 659, in __call__
    await route.handle(scope, receive, send)
  File "/opt/homebrew/Caskroom/miniforge/base/lib/python3.9/site-packages/starlette/routing.py", line 259, in handle
    await self.app(scope, receive, send)
  File "/opt/homebrew/Caskroom/miniforge/base/lib/python3.9/site-packages/starlette/routing.py", line 61, in app
    response = await func(request)
  File "/opt/homebrew/Caskroom/miniforge/base/lib/python3.9/site-packages/fastapi/routing.py", line 227, in app
    raw_response = await run_endpoint_function(
  File "/opt/homebrew/Caskroom/miniforge/base/lib/python3.9/site-packages/fastapi/routing.py", line 160, in run_endpoint_function
    return await dependant.call(**values)
  File "/Users/beginner/Projects/temp/fastapi-vue/backend/app/apps/permission/views.py", line 98, in getRole
    return respSuccessJson(curd_role.get(db, _id=role_id))
  File "/Users/beginner/Projects/temp/fastapi-vue/backend/app/apps/permission/curd/curd_role.py", line 28, in get
    res = role.dict()
AttributeError: Could not locate column in row for column 'dict'
WARNING 2022-07-20 15:58:16 StatReload detected file change in 'apps/permission/curd/curd_role.py'. Reloading...
INFO 2022-07-20 15:58:16 Shutting down
INFO 2022-07-20 15:58:16 Waiting for application shutdown.
INFO 2022-07-20 15:58:16 Application shutdown complete.
INFO 2022-07-20 15:58:16 Finished server process [25988]
INFO 2022-07-20 15:58:17 Started server process [26022]
INFO 2022-07-20 15:58:17 Waiting for application startup.
INFO 2022-07-20 15:58:17 Application startup complete.
WARNING 2022-07-20 15:59:25 StatReload detected file change in 'apps/permission/curd/curd_role.py'. Reloading...
INFO 2022-07-20 15:59:25 Shutting down
INFO 2022-07-20 15:59:25 Waiting for application shutdown.
INFO 2022-07-20 15:59:25 Application shutdown complete.
INFO 2022-07-20 15:59:25 Finished server process [26022]
INFO 2022-07-20 15:59:26 Started server process [26054]
INFO 2022-07-20 15:59:26 Waiting for application startup.
INFO 2022-07-20 15:59:26 Application startup complete.
WARNING 2022-07-20 16:00:30 StatReload detected file change in 'apps/permission/curd/curd_role.py'. Reloading...
INFO 2022-07-20 16:00:30 Shutting down
INFO 2022-07-20 16:00:30 Waiting for application shutdown.
INFO 2022-07-20 16:00:30 Application shutdown complete.
INFO 2022-07-20 16:00:30 Finished server process [26054]
WARNING 2022-07-20 16:00:38 StatReload detected file change in 'apps/permission/curd/curd_role.py'. Reloading...
INFO 2022-07-20 16:00:39 Started server process [26078]
INFO 2022-07-20 16:00:39 Waiting for application startup.
INFO 2022-07-20 16:00:39 Application startup complete.
WARNING 2022-07-20 16:17:21 StatReload detected file change in 'apps/permission/curd/curd_role.py'. Reloading...
INFO 2022-07-20 16:17:21 Shutting down
INFO 2022-07-20 16:17:21 Waiting for application shutdown.
INFO 2022-07-20 16:17:21 Application shutdown complete.
INFO 2022-07-20 16:17:21 Finished server process [26078]
INFO 2022-07-20 16:17:22 Started server process [26333]
INFO 2022-07-20 16:17:22 Waiting for application startup.
INFO 2022-07-20 16:17:22 Application startup complete.
WARNING 2022-07-20 16:33:16 StatReload detected file change in 'apps/permission/views.py'. Reloading...
INFO 2022-07-20 16:33:16 Shutting down
INFO 2022-07-20 16:33:16 Waiting for application shutdown.
INFO 2022-07-20 16:33:16 Application shutdown complete.
INFO 2022-07-20 16:33:16 Finished server process [26333]
INFO 2022-07-20 16:33:17 Started server process [26624]
INFO 2022-07-20 16:33:17 Waiting for application startup.
INFO 2022-07-20 16:33:17 Application startup complete.
WARNING 2022-07-20 18:42:34 StatReload detected file change in 'apps/permission/models/user.py'. Reloading...
INFO 2022-07-20 18:42:34 Shutting down
INFO 2022-07-20 18:42:34 Waiting for application shutdown.
INFO 2022-07-20 18:42:34 Application shutdown complete.
INFO 2022-07-20 18:42:34 Finished server process [26624]
INFO 2022-07-20 18:42:36 Started server process [28730]
INFO 2022-07-20 18:42:36 Waiting for application startup.
INFO 2022-07-20 18:42:36 Application startup complete.
WARNING 2022-07-20 19:09:01 StatReload detected file change in 'apps/permission/views.py'. Reloading...
INFO 2022-07-20 19:09:01 Shutting down
INFO 2022-07-20 19:09:02 Waiting for application shutdown.
INFO 2022-07-20 19:09:02 Application shutdown complete.
INFO 2022-07-20 19:09:02 Finished server process [28730]
INFO 2022-07-20 19:09:03 Started server process [29182]
INFO 2022-07-20 19:09:03 Waiting for application startup.
INFO 2022-07-20 19:09:03 Application startup complete.
WARNING 2022-07-20 19:18:55 StatReload detected file change in 'apps/permission/curd/curd_user.py'. Reloading...
INFO 2022-07-20 19:18:55 Shutting down
INFO 2022-07-20 19:18:55 Waiting for application shutdown.
INFO 2022-07-20 19:18:55 Application shutdown complete.
INFO 2022-07-20 19:18:55 Finished server process [29182]
INFO 2022-07-20 19:18:56 Started server process [29378]
INFO 2022-07-20 19:18:56 Waiting for application startup.
INFO 2022-07-20 19:18:56 Application startup complete.
WARNING 2022-07-21 11:19:59 StatReload detected file change in 'apps/permission/views.py'. Reloading...
INFO 2022-07-21 11:19:59 Shutting down
INFO 2022-07-21 11:19:59 Waiting for application shutdown.
INFO 2022-07-21 11:19:59 Application shutdown complete.
INFO 2022-07-21 11:19:59 Finished server process [29378]
INFO 2022-07-21 11:20:00 Started server process [31237]
INFO 2022-07-21 11:20:00 Waiting for application startup.
INFO 2022-07-21 11:20:00 Application startup complete.
WARNING 2022-07-21 12:23:44 StatReload detected file change in 'apps/permission/views.py'. Reloading...
INFO 2022-07-21 12:23:44 Shutting down
INFO 2022-07-21 12:23:44 Waiting for application shutdown.
INFO 2022-07-21 12:23:44 Application shutdown complete.
INFO 2022-07-21 12:23:44 Finished server process [31237]
INFO 2022-07-21 12:23:45 Started server process [32369]
INFO 2022-07-21 12:23:45 Waiting for application startup.
INFO 2022-07-21 12:23:45 Application startup complete.
INFO 2022-07-21 16:05:56 Shutting down
INFO 2022-07-21 16:05:57 Waiting for application shutdown.
INFO 2022-07-21 16:05:57 Application shutdown complete.
INFO 2022-07-21 16:05:57 Finished server process [32369]
